data/hai/hai reveal 5 philosophers.txt

```diff
@@ -319,6 +402,7 @@ mission "Hai Reveal [B05-short] The Road Not Taken"
 		has "Hai Reveal [B04] The Road Not Taken: done"
```

Comment:
I can't comment on line 395, but the description isn't in line with other ones. I'd keep Monopoly out of this.

Reply:
It is removed.